src/commands/perform-tasks.ts

```diff
@@ -47,6 +47,7 @@ export class PerformTasksCommand extends BaseCliCommand<IPerformTasksCommandArgs
         command.option('--templating-context-file [templating-context-file]', 'json file used as context for nunjuck text templating of organization and tasks file');
         command.option('--match [match]', 'glob pattern used to define/filter which tasks to run.');
         command.option('--large-template-bucket-name [large-template-bucket-name]', 'bucket used when uploading large templates. default is to create a bucket just-in-time in the target account');
+        command.option('--dev-role [dev-role]', 'use the development build role that does not have access to production environments.', true);
```

Comment:
I would like to think a bit longer about the name of this option (doesn't affect the design in any way though!)
--dev-role seems to suggest you can pass a dev-role
maybe something like '--use-dev-role' or '--dev'
I think I like --dev as we can also have other behaviour changes when passing in that flag.